deeplabcut/pose_estimation_tensorflow/core/evaluate.py

```diff
@@ -139,13 +135,17 @@ def calculatepafdistancebounds(
                                     (Data[ind, j1, "x"] - Data[ind2, j2, "x"]) ** 2
                                     + (Data[ind, j1, "y"] - Data[ind2, j2, "y"]) ** 2
                                 )
-                                / dlc_cfg["stride"]
+                                / test_pose_cfg["stride"]
                             )
                         else:
                             distances = None
 
                         if distances is not None:
                             if onlytrain:
+                                train_pose_cfg = load_config(str(path_train_config))
+                                _, trainIndices, _, _ = auxiliaryfunctions.load_metadata(
+                                    Path(cfg["project_path"]) / train_pose_cfg['metadataset']
+                                )
```

Comment:
I'd take this out of the loop to avoid loading the metadata over and over again.